extension/source/background/AggregatorController.ts

```diff
@@ -126,21 +123,10 @@ export default class AggregatorController {
 
       const aggregator = new Aggregator(knownTx.aggregatorUrl);
 
-      // FIXME: Aggregator/bls-wallet-clients: The response we're getting is
-      // different from the type annotation.
-      const bundleReceipt: unknown = await aggregator.lookupReceipt(hash);
-
-      assertType(
-        bundleReceipt,
-        optional(
-          io.type({
-            transactionIndex: io.number,
-            transactionHash: io.string,
-            blockHash: io.string,
-            blockNumber: io.number,
-          }),
-        ),
-      );
+      const bundleReceiptResponse = await aggregator.lookupReceipt(hash);
+      // If receipt doesn't exist, set to undefined to be consistent with
+      // the return type
+      const bundleReceipt = bundleReceiptResponse?.receipt || undefined;
```

Comment:
`|| undefined` is unnecessary (`obj?.field` produces `undefined` when field is not there, even when `obj` is `null`).

Also prefer `??` over `||`.

Reply:
Yeah I was getting a typescript error the `BundleReceipt | undefined` does not equal `BundleReceipt | undefined | null`.  I think with the changes you recommended above I can fix this.